src/shared/containers/app.jsx

```diff
@@ -161,6 +164,10 @@ class App extends React.Component {
     let toolbox;
     let fab;
     if (currentScreen) {
+      if (currentScreen.displayToolbar != null) {
```

Comment:
could be undefined instead of null ?

Reply:
yes this is why I used `!=` and not `!==`. Using `!=` will manage case where `displayToolbar` is null or undefined

https://jsfiddle.net/xomq737q/3/